app/core/contact.coffee

```diff
@@ -37,3 +37,10 @@ module.exports =
       data: {teacherEmail, sessionId, levelName, codeLanguage: _.string.titleize(codeLanguage).replace('script', 'Script')}
     })
     return new Promise(jqxhr.then)
+
+  sendParentTeacherSignup: ({teacherEmail, parentEmail, parentName, customContent}) ->
```

Comment:
The spec has changed and only the custom method is used. Is it still useful to send all the data in case the spec requires it in the future?